<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelVitesse">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XVitesse</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRegime">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XRegime</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBoite">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XBV</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEtat">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XEtat Moteur</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTempEau">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XTemp Eau</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCapEau">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XCap Eau</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCapHuile">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>XCap Huile</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCapAir">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>XCap Air</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
